### PR TITLE
Vel101 432 create a case in salesforce

### PR DESCRIPTION
**Pull request by lawrencec**
This PR, [VEL101-432](https://livesport.atlassian.net/browse/VEL101-432),  creates a Case in Salesforce.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -84,7 +84,9 @@ pipeline {
         TF_VAR_environment="develop"
         TF_VAR_image_uri="foobar" // hardcoded values just for the plan purpose
         TF_VAR_image_tag="fizzbuzz" // hardcoded values just for the plan purpose
```
Comment (clickbuster) [inline Jenkinsfile]: I don't think you need those anymore

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "test:unit": "NODE_ENV=test jest ./test/unit",
     "test": "NODE_ENV=test jest ./test",
     "test:watch": "npm test -- --watch",
-    "coverage": "npm run test:unit --coverage",
+    "coverage": "npm run lint && npm run test:unit --coverage",
```
Comment (clickbuster) [inline package.json]: I'm not entirely sure, but should the last bit be `npm run test:unit -- --coverage`?

Comment (lawrencec) [reply package.json]: Good shout. Jest is already configured to collect coverage so this task is not required any more.

File: Jenkinsfile (hunk 1 above)
Comment (lawrencec) [reply Jenkinsfile]: Oops, looks like they are required @clickbuster.

http://jenkins3.dazndev.com/blue/organizations/jenkins/help-api-contact-us/detail/PR-3/6/pipeline

**Pull request merged**